pages/doc/proxies_preprocessor_rules.md

```diff
@@ -301,7 +301,7 @@ Enables you to pass only metrics that are provided in a `preprocessor_rules.yaml
 </tr>
 <tr>
 <td>names</td>
-<td>List of metric names or regular expressions. The list specifies which metrics are sent to the Wavefront service or which are dropped.
+<td>List of exact metric names or regular expressions. The list specifies which metrics are sent to the Wavefront service or which are dropped.
```

Comment:
We should clarify that regular expression have to start with a "/" and ends with a "/"

Reply:
Just pushed a change there - let me know what you think.